markdown/ch5.markdown

```diff
@@ -577,14 +572,12 @@ were head of the NumPy Agency for Space Affairs and had to rotate
 millions of images streaming in from the newly launched Jupyter
 Orbiter?
 
-Of course, one option would be to rewrite all your code in C++, but
-let's presume you have diligently read Scott Meyers' excellent books
-and are deeply convinced that you will never write bug free C++ under
-pressure.  While options like Cython, Numba, or Julia are available,
-we'd like to show you a quick workaround using SciPy's sparse
-matrices.
+In such cases, you want to squeeze every ounce of performance from your
```

Comment:
The comment was that the intro was overly complicated. My own take is that

a) a very tiny minority of our readers would have thought of Cython/Numba/Julia, and a much tinier still minority would have thought of C++. So it felt a bit jarring to me to mention all that.
b) at any rate, SciPy's functions are implemented highly optimised C, so it's unclear why we should be able to speed them up.
c) we never mentioned that the thing we can make fast is repeated application of the same transform. Now we do.